### ISSUE TITLE
tidal voicings?

### ISSUE THREAD

**Issue by felixroos** [tidalcycles/strudel#505]
it might make sense to add a voicing dictionary with chord symbols that match tidals standard, see https://tidalcycles.org/docs/patternlib/howtos/playchords/#loading-the-chord-list

so you could write:

```js
"Dmin7 Gdom7 Cmaj7@2".voicings('tidal')
```

**Comment by felixroos** [comment]
some things cannot be implemented just with voicings, for example, c'min'7 will stack 7 notes from cmin (not cmin7). e.g. `c3'min'7` would give `c3 eb3 g3 c4 eb4 g4 c5`

**Comment by eefano** [comment]
but c3'min7 can

**Comment by felixroos** [comment]
> but c3'min7 can

yes, i meant that the stacking feature is not possible with dictionary voicings
